tutorials/lbs/first/first_example.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sun, February 23
+
+Copyright (c) 2025 quocdang1998
+"""
+
+import os
+import sys
+import math
+
+sys.path.append(os.path.abspath("../../../"))
+
+from pyopensn.mesh import OrthogonalMeshGenerator, KBAGraphPartitioner
+from pyopensn.mat import AddMaterial
+from pyopensn.xs import LoadFromOpenSn
+from pyopensn.source import VolumetricSource
+from pyopensn.aquad import ProductQuadrature
+from pyopensn.solver import DiscreteOrdinatesSolver, SteadyStateSolver
+from pyopensn.fieldfunc import FieldFunctionGridBased
+
+if __name__ == "__main__":
+
+    # compute node coordinates
+    nodes = []
+    n_cells = 10
+    length = 2.0
+    xmin = - length / 2
+    dx = length / n_cells
+    for i in range(n_cells + 1):
+        nodes.append(xmin + i * dx)
+
+    # generate the mesh
+    meshgen = OrthogonalMeshGenerator(
+        node_sets = [nodes, nodes],
```

Comment:
As for the python formatting, we have `.flake8` file in the root repo. If you install `flake8`, you can run:
```
cd /path/to/repo
flake8 tutorials
```

And it will tell you locally the problems you need to fix.